libraries/adaptive-expressions/src/functionUtils.ts

```diff
@@ -0,0 +1,1033 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import {TimexProperty} from '@microsoft/recognizers-text-data-types-timex-expression';
+import * as lodash from 'lodash';
+import moment, {Moment} from 'moment';
+import {Constant} from './constant';
+import { ReturnType } from './returnType';
+import {EvaluateExpressionDelegate} from './expressionEvaluator';
+import {ExpressionType} from './expressionType';
+import {convertCSharpDateTimeToMomentJS} from './datetimeFormatConverter';
+import {MemoryInterface, SimpleObjectMemory, StackedMemory} from './memory';
+import {Options} from './options';
+import bigInt = require('big-integer');
+
+/**
+ * Verify the result of an expression is of the appropriate type and return a string if not.
+ * @param value Value to verify.
+ * @param expression Expression that produced value.
+ * @param child Index of child expression.
+ */
+export type VerifyExpression = (value: any, expression: any, child: number) => string | undefined;
```

Comment:
>any [](start = 56, length = 3)

Again all of the :any for Expression in this file seem like a bad idea.  Why not use : Expression?